doc/ics/yaml/merton_gov_uk.yaml

```diff
@@ -0,0 +1,13 @@
+---
+title: London Borough of Merton (New)
+url: https://www.merton.gov.uk
+howto:
+  en: |
+    - Go to <https://fixmystreet.merton.gov.uk/waste> and enter your post code, and on the following page select your address.
+    - In the Download your collection schedule panel, click Add to your calendar.
+    - Click the Copy button in Step 1: copy the link to the calendar
+    - Replace the `url` in the example configuration with the link that has just been copied to your clipboard.
```

Comment:
>     - Replace the `url` in the example configuration with the link that has just been copied to your clipboard.

I don't like this phrase anymore as it will be displayed in the configuration GUI where there is no example

Reply:
I don't see any text like that when setting up ics sources in the GUI. Does HA pull text from the yaml files, or just the GUI dicts added to py source files?
![image](https://github.com/user-attachments/assets/f9785f4f-53c2-4b77-9705-28965fc82c0c)

An alternative text could be something like: _Paste the copied url into the GUI configuration dialog for the Generic|ICS source,  or your config.yaml file_

Reply:
Adding the YAML files adds links in the GUI so people find their region in their country section. If you select the link, you see the how-to text (that's why it can be translated now). The update_docu_links.py puts the text in the translations files.

I use something like "Use this URL as the `url` parameter." now.

![image](https://github.com/user-attachments/assets/4f8b95d2-0e81-4749-a1e3-d11bc75cc049)
![image](https://github.com/user-attachments/assets/5306527c-0f42-417d-acba-1292c4f1d575)

Reply:
yaml text updated